File: mobile/src/app/models/homerecentreview.dto.ts
export interface HomeRecentReviewDto {
  reviewId: number;
  perfumeId: number;

  perfumeName: string;
  brand: string;
  perfumeImageUrl: string | null;

  author: string;
  rating: number;
  comment: string;
  createdAt: string;
}

File: backend/Core/DTOs/HomeRecentReviewDto.cs
namespace Core.DTOs
{
    public sealed class HomeRecentReviewDto
    {
        public int ReviewId { get; init; }
        public int PerfumeId { get; init; }
        public string PerfumeName { get; init; } = null!;
        public string Brand { get; init; } = null!;
        public string? PerfumeImageUrl { get; init; }
        public string Author { get; init; } = null!;
        public int Rating { get; init; }
        public string Comment { get; init; } = null!;
        public DateTime CreatedAt { get; init; }
    }
}
